src/registry/addressNL/edit.tsx

```diff
@@ -19,12 +21,71 @@ import {
 } from '@/components/builder';
 import {LABELS} from '@/components/builder/messages';
 import {Checkbox} from '@/components/formio';
-import {TabList, TabPanel, Tabs} from '@/components/formio';
+import {DataMap, Panel, Tab, TabList, TabPanel, Tabs} from '@/components/formio';
 import {Select} from '@/components/formio';
+import {BuilderContext} from '@/context';
 import {useErrorChecker} from '@/utils/errors';
 
 import {EditFormDefinition} from '../types';
 
+export interface SubcomponentValidationProps {
+  component: string;
```

Comment:
```suggestion
  component: 'postcode' | 'city'; // or even `keyof Required<AddressNLComponentSchema>['openForms']['components']` or something like that :)
```
try to keep types as narrow as possible - we know upfront what does and doesn't exist, let's make the type system double check against typos etc.